addons/sale/wizard/sale_payment_link.py

```diff
@@ -35,6 +45,7 @@ def _generate_link(self):
                                     f'?reference={urls.url_quote(payment_link.description)}' \
                                     f'&amount={payment_link.amount}' \
                                     f'&sale_order_id={payment_link.res_id}' \
+                                    f'{"&acquirer_id=" + str(payment_link.payment_acquirer_id.id) if payment_link.has_multiple_acquirer and payment_link.payment_acquirer_id.id else "" }' \
```

Comment:
Is the check on `has_multiple_acquirer` useful here ?
The `acquirer_id` cannot be specified if there aren't multiple acquirers (and if it is, it's a custo and not really a problem to specify the acquirer)